addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2496,7 +2517,13 @@ export class OdooEditor extends EventTarget {
             const selection = this.document.getSelection();
             if (selection.isCollapsed) {
                 ev.preventDefault();
-                this._applyCommand('oDeleteBackward');
+                const inputEvent = new InputEvent('input', {
```

Comment:
Why do we need to simulate an event and not call directly `oDeleteBackward`?

Reply:
It ensure all delete backward follow the same route ( aka the input handler) So we only have to ensure this route is bug free and not both route.

Reply:
But it's because there was an issue with calling `oDeleteBackward` directly from here, right ?

Reply:
It was not an issue, it was hiding another issue that was only visible in mobile. because mobile keyboard already used the input handler route.
But this show that we could also have issue with calling `oDeleteBackward` directly.

At least now we reduce the potential "hidden" errors.

Reply:
So basically this is a fix for mobile, right ?

Reply:
yes

Reply:
Could you please move it in its own commit then ?